apps/AEPSampleApp/extensions/EdgeIdentityView.js

```diff
@@ -78,31 +79,4 @@ function removeIdentity() {
   
   console.log("sample app - removeIdentity");
   Identity.removeIdentity(item1, namespace);
-}
-
-
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center',
-    backgroundColor: '#F5FCFF',
-  },
-  welcome: {
-    fontSize: 22,
-    textAlign: 'center',
-    margin: 10,
-  },
-  text: {
```

Comment:
Nice, let's also move text and breakLine to styles so we can reuse them in the other extensions